src/routes/Messages/Messages.js

```diff
@@ -24,9 +25,17 @@ export default class Messages extends React.Component {
   constructor (props) {
     super(props)
 
+    let messagesOpen = false
+    if (props.messageThreadId && !props.isInbox) {
+      messagesOpen = true
+    } else if (props.isInbox) {
```

Comment:
you dont need this condition since it is already set to false above

Reply:
k